devel/addon/sample-external-issuer/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@io_bazel_rules_docker//container:bundle.bzl", "container_bundle")
+
+container_bundle(
+    name = "bundle",
+    images = {
+        "ghcr.io/wallrj/sample-external-issuer/controller:v0.0.0-30-gf333b9e": "@io_ghcr_wallrj_sample-external-issuer_controller//image",
```

Comment:
Can you file an issue to move this to quay once we have access to our org

Reply:
Done. See https://github.com/jetstack/cert-manager/issues/3531